server/src/database/migrations/postgres/004-instance-runners.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN TRANSACTION;
```

Comment:
Remember to check that migration number does not clash with other new migrations from different branches